Showcase_Apps/README.md

```diff
@@ -10,6 +10,7 @@ All of the applications in these folders are full working applications.  Please
 * Image Handling
 * PDF Generation
 * Printing and QR Codes
+* Video Player View Extension App
 * Multi-Select & Context Menu App
```

Comment:
@robinkuck  can you please re-arrange list like below?

* Image Handling
* Multi-Select & Context Menu
* PDF Generation
* Persistent Filters
* Printing and QR Codes
* Extension Samples
   * Calender View
   * Video Player